### ISSUE TITLE
Cash Points not Updating realtime ( 1.88f5 )

### ISSUE THREAD

**Issue by lunidel** [suriyun-production/mmorpg-kit-docs#2499]
Cash points do not update in real time ( tested with config both database cache on/off )

![image](https://github.com/suriyun-production/mmorpg-kit-docs/assets/100618103/69d38379-9cf9-4515-a3b1-06db6efcd385)

![image](https://github.com/suriyun-production/mmorpg-kit-docs/assets/100618103/16007195-09b1-4b3f-a5c1-52a3c6c8d8d6)

However if you buy something to change your current value, it applies almost immediately. 

![image](https://github.com/suriyun-production/mmorpg-kit-docs/assets/100618103/83cf09df-446a-4dec-b6d8-8865317a6b66)

**Comment by insthync** [comment]
I won't fix it, I actually want to force everyone to use caching

**Comment by insthync** [comment]
If you want to do it, then just request for cash amount by any number of interval and use it to update character entity cash amount, that is it.

**Comment by lunidel** [comment]
What I meant was I tested it with both off and on settings and result is the same. @insthync

**Comment by lunidel** [comment]
I always use caching since last version.

**Comment by insthync** [comment]
Yes it is working as intended, that is how it must be, it is never real time

**Comment by lunidel** [comment]
But it only updates when servers are restarted or when they purchase something which is hard if they start with 0

**Comment by insthync** [comment]
I've told you how to do it as you want. I won't change anything which will affect everyone just for you only.
